src/types/package.rs

```diff
@@ -567,7 +567,8 @@ pub struct PackageStatus {
     pub num_dependencies: u32,
     /// Number of vulnerabilities found in this package and all transitive
     /// dependencies
-    pub num_vulnerabilities: u32,
+    #[serde(default)]
+    pub num_vulnerabilities: Option<u32>,
```

Comment:
Just to be clear, this struct will need to be updated again to remove this value when we are ready to make the change in API...

As written, it would serialize `None` as `"num_vulnerabilities": null`, which isn't great.